sdk/cosmos/azure-cosmos/azure/cosmos/diagnostics.py

```diff
@@ -80,6 +85,10 @@ def __call__(self, headers, body):
         self._request_charge += float(headers.get("x-ms-request-charge", 0))
 
     def __getattr__(self, name):
+        warnings.warn("The `diagnostics.py` file is deprecated and will be removed in the future. For logging " +
```

Comment:
I thought we wanted to add this to the module-level getattr?

Reply:
No. This current warning will only print when I access an attribute on the RecordDiagnostics model. 
We would want to overwrite the module-level `__getattr__` function so that when this model is imported the warning is printed.

Reply:
ahhh I see what you mean, just pushed the changes!

Reply:
Nevermind - I tried doing this after reading this link: https://stackoverflow.com/questions/2447353/getattr-on-a-module but it seems like the CI pipelines don't like it. Should I go about it the hacky way being described in the post?